plugins/httpd-client.ts

```diff
@@ -0,0 +1,20 @@
+export default defineNuxtPlugin(() => {
+  const {
+    public: { openFetch: clients },
+  } = useRuntimeConfig()
+
+  return {
+    provide: Object.fromEntries(
+      Object.entries(clients).map(([name, _client]) => [
+        `${name}Fetch`,
+        createOpenFetch((options) => ({
+          ...clients.httpd,
+          ...options,
+          async onRequest(ctx) {
+            return await options.onRequest?.(ctx)
+          },
```

Comment:
```suggestion
```

This seems like a passthrough without the `console.log()`. I think we can do away with it altogether.

Reply:
Seems like it, removed.